meson.build

```diff
@@ -58,6 +58,13 @@ else
   dxvk_library_path = meson.source_root() + '/lib32'
 endif
 
+# Set define for arch
+if (cpu_family == 'x86' or cpu_family == 'x86_64')
+  add_project_arguments('-DCPU_ARCH_X86', language : ['c', 'cpp'])
```

Comment:
Probably define both x86_64 too?

Or rather, why not just use ``__x86_64__`` and friends?

Reply:
I rather prefer meson to provide us the arch to target than having to look each compiler define's for archs.
Also the defines are used only to have x86/x86_64 or ARM/Aarch64 specific code, not between 32 or 64 bits.